src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/MethodologyContentPage.tsx

```diff
@@ -38,33 +46,122 @@ const MethodologyContentPage = ({
           isCommenting: false,
           isReviewing: false,
           releaseId: methodology.id,
           availableDataBlocks: [],
         }}
       >
-        <main
-          className="govuk-main-wrapper app-main-class"
-          id="main-content"
-          role="main"
+        <div
+          className={classNames('govuk-width-container', {
+            'dfe-align--comments': isEditing,
+            'dfe-hide-comments': !isEditing,
+          })}
         >
-          <h1
-            className="govuk-heading-xl"
-            data-testid={`page-title ${methodology.title}`}
-          >
-            {methodology.title}
-          </h1>
-          <p>Lorem ipsum.</p>
-          <p>{JSON.stringify(methodology)}</p>
-        </main>
+          <section className={isEditing ? 'page-editing' : 'page-preview'}>
+            <h1
+              className="govuk-heading-xl"
+              data-testid={`page-title ${methodology.title}`}
+            >
+              {methodology.title}
+            </h1>
+
+            <div className="govuk-grid-row">
+              <div className="govuk-grid-column-two-thirds">
+                <dl className="dfe-meta-content govuk-!-margin-top-0">
+                  <div>
+                    <dt className="govuk-caption-m">Published: </dt>
+                    <dd data-testid="published-date">
+                      <strong>
+                        {methodology.published ? (
+                          <FormattedDate>{methodology.published}</FormattedDate>
+                        ) : (
+                          'Not yet published'
+                        )}
+                      </strong>
+                    </dd>
+                  </div>
+                  {methodology.lastUpdated &&
+                    methodology.lastUpdated.length > 0 && (
+                      <>
+                        <dt className="govuk-caption-m">Last updated: </dt>
+                        <dd data-testid="last-updated">
+                          <strong>
+                            <FormattedDate>
+                              {methodology.lastUpdated}
+                            </FormattedDate>
+                          </strong>
+                        </dd>
+                      </>
+                    )}
+                </dl>
+                {!isEditing && (
+                  <>
+                    <PrintThisPage
+                      analytics={{
+                        category: 'Page print',
+                        action: 'Print this page link selected',
+                      }}
+                    />
+                    <PageSearchForm inputLabel="Search in this methodology page." />
+                  </>
+                )}
+              </div>
+            </div>
+
+            <Accordion
+              id="contents-accordion"
+              canReorder
+              sectionName="Contents"
+              onSaveOrder={order => {
+                return new Promise(resolve => {
```

Comment:
Use async/await. This is currently dangerous, as if there is actually an error, the wrapping promise will never resolve.